aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -170,6 +171,17 @@
    */
   public static final String S3FILEIO_SESSION_TOKEN = "s3.session-token";
 
+  /**
+   * Enable to make S3FileIO, to make cross-region call to the region specified in the ARN.
+   * <p>
+   * When set, the client can make a cross-region call to the region specified in the ARN,
+   * otherwise an exception will be thrown if cross-region call is tried.
+   *
+   * For more details see : https://github.com/aws/aws-sdk-java-v2/blob/master/services/s3/src/main/java/software/amazon/awssdk/services/s3/S3Configuration.java#L186
+   */
+  public static final String S3_ARN_REGION_ENABLED = "s3.arn-region-enabled";
```

Comment:
Sorry I just notice the name in S3Configuration is `useArnRegionEnabled`, I think we should map the name exactly to the s3 to call it `s3.use-arn-region-enabled`